reference/docs-conceptual/dsc/getting-started/winGettingStarted.md

```diff
@@ -68,45 +68,63 @@ Configuration EnvironmentVariable_Path
 
 EnvironmentVariable_Path -OutputPath:"C:\EnvironmentVariable_Path"
 ```
+
 #### Install a module containing DSC resources
 
 Windows PowerShell Desired State Configuration includes built-in modules containing DSC resources.
-You can also load modules from external sources such as the PowerShell Gallery, using the PowerShellGet cmdlets.
+You can also load modules from external sources such as the PowerShell Gallery, using the
+PowerShellGet cmdlets.
 
-`Install-Module 'PSDscResources' -Verbose`
+```PowerShell
+Install-Module 'PSDscResources' -Verbose
+```
 
 #### Apply the configuration to the machine
 
-Configuration documents (MOF files) can be applied to the machine
-using the [Start-DscConfiguration](/powershell/module/psdesiredstateconfiguration/start-dscconfiguration) cmdlet.
+> [!NOTE] To allow DSC to run, Windows needs to be configured to receive PowerShell remote commands,
```

Comment:
Reflow wrapped this incorrectly.

Reply:
Followed the errors there and wondered how in the world I did this and then remembered. Sigh, live and learn